### ISSUE TITLE
Unknown command: current_hero

### ISSUE THREAD

**Issue by Silox** [loopuleasa/Dota2-TheCore-Config-Engine#130]
Every time a modifier is used, the console spits out `Unknown command: current_hero`, or variations: `
Unknown command: current_hero_alt` and `Unknown command: current_hero_space`

**Comment by loopuleasa** [comment]
Yeah, that is something that appears because no current_hero custom mode is set. 

I am unsure how to remove them while still keeping functionalities. If anyone has a solution please let me know.

**Comment by dodekeract** [comment]
@loopuleasa Can't you just add a fake alias to the config files, which are always loaded?

**Comment by Sembrani** [comment]
I will have a look into it.

**Comment by loopuleasa** [comment]
Is the issue still in after the Arc Warden commit?
